Automattic/themes
# Modern Business: Full Site Editing - add header and footer selectors for block front end styles

#### Changes proposed in this Pull Request:

Add  `.fse-template-content` class to block styles in order to apply the correct front end block styles to the FSE header and footer content.

#### Testing
- Check out https://github.com/Automattic/wp-calypso/pull/35154 in your FSE test environment
- Build theme and copy or link to local FSE install
- Add a Paragraph block to the Header and Footer template parts and set the background colour to one of the primary colours
- View the page in the front end and check that the blocks have a background colour

**Before**

<img width="534" alt="before" src="https://user-images.githubusercontent.com/3629020/62341603-1369af00-b538-11e9-8108-95633455e74d.png">

**After**

<img width="519" alt="after" src="https://user-images.githubusercontent.com/3629020/62341613-1cf31700-b538-11e9-9d65-f6e237322f8e.png">

#### Related issue(s): 
https://github.com/Automattic/wp-calypso/issues/35058
https://github.com/Automattic/wp-calypso/pull/35154
